lib/evmone/baseline.cpp

```diff
@@ -276,7 +323,14 @@ evmc_result execute(const VM& vm, ExecutionState& state, const CodeAnalysis& ana
     if (INTX_UNLIKELY(tracer != nullptr))
         dispatch<true>(cost_table, state, tracer);
     else
-        dispatch<false>(cost_table, state);
+    {
+#if EVMONE_CGOTO_SUPPORTED
```

Comment:
We can eliminate the `#if` by providing a stub `dispatch_cgoto()` implementation for unsupported compilers and making sure `vm.cgoto` is always false. However, the last condition can only be checked at runtime. With the current version we will get compilation error if something is messed up.